pom.xml

```diff
@@ -9,7 +9,7 @@
     <relativePath />
   </parent>
   <artifactId>strict-crumb-issuer</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>${changelist}</version>
```

Comment:
Why did you decide against a prefix?

Reply:
Simply because that's the recommended way from the [documentation](https://www.jenkins.io/doc/developer/publishing/releasing-cd/#configure-cd-workflow), is there a need to have a prefix?

Reply:
No need, but IMO it's better:

* It retains the option of returning from CD versioning without a ridiculous version number. This has already happened to a few plugins who now have ridiculous version numbers that lack the semantic meaning from CD versioning.
* Dependabot interprets major version numbers greater than 1900 as years, so if this plugin ever gets up there, it may cause unintentional downstream effects (unlikely, but still, why provoke this?)